mongo/integration/client_test.go

```diff
@@ -472,3 +474,108 @@ type proxyMessage struct {
 	sent          wiremessage.WireMessage
 	received      wiremessage.WireMessage
 }
+
+func TestClientStress(t *testing.T) {
+	// TODO: Enable with GODRIVER-2038.
+	t.Skip("TODO: Enable with GODRIVER-2038")
+
```

Comment:
We may want to call [`Integration()`](https://github.com/mongodb/mongo-go-driver/blob/master/internal/testutil/config.go#L321-L328) here to skip the integration test if we are in short testing mode.

Reply:
Seems like a good idea. It looks like we don't use `testutil.Integration` anywhere currently. Are there use cases currently for running the tests in short mode?

Reply:
Interesting; there are no occurrences of `testutil.Integration` so far, but we do check `testing.Short()` in [`polling_srv_records_test`](https://github.com/mongodb/mongo-go-driver/blob/master/x/mongo/driver/topology/polling_srv_records_test.go#L128-L130). We also have a `test-short` Makefile [target](https://github.com/mongodb/mongo-go-driver/blob/master/Makefile#L82-L86), but I don't believe it's ever used in CI... 

I'll leave it up to you but might as well include a `testutil.Integration()` call since this test seems to be relatively time-intensive.

Reply:
I'll update the `testing.Short()` checks in `polling_srv_records_test.go` to use `testutil.Integration()` and add that to `TestClientStress` as well.

Reply:
Updated.